docker_compose/compose.yaml

```diff
@@ -0,0 +1,52 @@
+# To start, run:
+# % docker compose up
+#
+# What you should see:
+#   1. lots of waiting while containers are downloaded (or nothing if cached)
+#   2a. crawler should run and download 1 sample pdf
+#   2b. demo ui starting (in parallel with 2a)
+#   3. lots of logging as opensearch get started eventually 'Waiting for opensearch to exit.'
```

Comment:
I don't think logs go to stdout by default?

Reply:
If you run the command as shown they did for Alex, Parth, and me.
If you run it with -d (detach) they don't.